### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by Amith225** [Amith225/NPerlinNoise#7] [General]
This is the start of general discussion

**Comment by shoffer** [comment]
I do not want to open as an issue since it might be user error. 
I have tried running examples listed after install in pycharm with Python 3.12.3 and at commandline with Python 3.10.
In both instances I get the error:
from nPerlinNoise import LinearColorGradient, meshgrid
ModuleNotFoundError: No module named 'nPerlinNoise'

edit: Running on Windows 10

**Comment by amithm3** [comment]
@shoffer thats a known issue #24 
please refer https://github.com/amithm3/nPerlinNoise?tab=readme-ov-file#note-if-the-package-is-not-being-imported-try-import-nperlinnoise-as-npn-refer-24 to solve it

**Comment by amithm3** [comment]
use `import NPerlinNoise as nPn`
